### PR TITLE
Added methodology page link on dashboard

### LINKED ISSUE #367: Methodology link missing on dashboard page
#### Description
We have a page dedicated to the methodology of how the test that are shown on the dashboard are run the link to the webpage is :[ smp-spec.io/dashboard/methodology](https://smp-spec.io/dashboard/methodology). Currently we don't have a way to go to this page from the dashboard page.

#### Expected Behavior
Add a way to go to this link from the [dashboard](https://smp-spec.io/dashboard). We can add a heading on the vertical menu shown below, that links to the [methodology page](https://smp-spec.io/dashboard/methodology)

#### Screenshots
image

#### Enviroment:
 - OS: [e.g. Ubuntu]
 - Browser: [e.g. Chrome, Safari]
 - Version: [e.g. 22]
 - Device: [e.g. laptop, iPhone 8]

---
### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Adi-ty**
Signed-off-by: Aditya <adityasinghboss1234@gmail.com>

This PR adds the methodology page link in the sidebar of dashboard page

This PR fixes #367 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by asubedy — CHANGES_REQUESTED** [review]
Let's center the methodology header and add underline on hover

**Comment by Adi-ty** [comment]
I have made the changes please review and merge if everything is good

File: docs/_includes/sidebar.html [hunk 1]
```diff
@@ -50,4 +50,13 @@
         <i>JWT Rewrite</i>
       </li>
     </ul>
-  </div>
\ No newline at end of file
+
+   <ul class="nav flex-coloumn" style="padding-top:1rem; margin-left:.5rem;">
+     <a class='newlink'
+     style="font-size:1.1rem; margin: auto;"
```
Comment (asubedy) [inline docs/_includes/sidebar.html:56]:
```suggestion
     style="font-size:1.3rem; margin-top: 2rem;"
```

**Review by asubedy — CHANGES_REQUESTED** [review]

**Review by asubedy — APPROVED** [review]
LGTM @Adi-ty 🚀

**Pull request merged**